test/common.py

```diff
@@ -2049,23 +2049,39 @@ class TestServerHandler(SimpleHTTPRequestHandler):
     # Request header handler for default do_GET() path in
     # SimpleHTTPRequestHandler.do_GET(self) below.
     def send_head(self):
+      path = self.translate_path(self.path)
+      try:
+        f = open(path, 'rb')
+        fs = os.fstat(f.fileno())
+      except IOError:
+        self.send_error(404, f'File not found {path}')
+        return None
       if self.path.endswith('.js'):
```

Comment:
Any idea why `.js` files are singled out here?

Reply:
Not sure, it was like that before: https://github.com/emscripten-core/emscripten/blob/3084681dd8b8f834a0d9c4cc034f90c0ad4937b1/test/common.py#L2055

But I removed the special case for JS and the tests seem to work fine.